scripts/qa_crowbarsetup.sh

```diff
@@ -3573,7 +3573,7 @@ function oncontroller_magnum_service_setup
     local service_image_name=magnum-service-image
     local service_image_url=http://clouddata.cloud.suse.de/images/$arch/other/${service_image_name}.qcow2
     local service_sles_image_name=sles-openstack-magnum-kubernetes
-    local service_sles_image_url=http://download.suse.de/ibs/Devel:/Docker:/Images:/SLE12SP1-JeOS-k8s-magnum/images/${service_sles_image_name}.${arch}.qcow2
+    local service_sles_image_url=http://download.suse.de/ibs/Devel:/Docker:/Images:/SLE12SP2-JeOS-k8s-magnum/images/${service_sles_image_name}.${arch}.qcow2
```

Comment:
could we have the URL use $susedownload instead of hardcoding the hostname?
same for $clouddata above, even though it is somewhat unrelated to this PR.